### PR TITLE
Improve the artifacts finder plugin.

### PR DESCRIPTION
**Pull request by snowtigersoft**
1. Change the prospect interval to 5 minutes and find will happen in 10 seconds after prospect finish.
2. Add concurrency respect actions to 10(can change the MAX_PROSPECTING variable easily)

### CODE REVIEW

**Review by Stx69 — APPROVED** [review]

File: content/artifacts/artifacts-finder/plugin.js [hunk 1]
```diff
@@ -56,54 +60,113 @@ function createDom(tag, text) {
   return dom;
 }
 
+function sleep(ms) {
```
Comment (jacobrosenthal) [inline content/artifacts/artifacts-finder/plugin.js:63]: This is .. not great. Theres definitely a way to wait for tx, but it is complicated and were pushing for a refactor so probably fine for now

**Review by jacobrosenthal — APPROVED** [review]
Thanks!

File: content/artifacts/artifacts-finder/plugin.js (hunk 1 above)
Comment (snowtigersoft) [reply content/artifacts/artifacts-finder/plugin.js:63]: Yes, sleep is not good. I'll try to find a new solution to do this.

**Comment by Stx69** [comment]
Will be nice one small cosmetic edit for "Deposit" screen, I try found how to add column rarity: ArtifactRarity ; to see 
image - but I cant found declaration via Icons.js somethink like function extraction ArtifactRarityLabel missing. Second small think for is "Untaken" screen if is activated - enable function first "by touch on map" or "selection of planet" = rewrite home x , y for sorting.

**Comment by jacobrosenthal** [comment]
@Stx69 perhaps you can create a new issue for that feature request?

**Pull request merged**